noxfile.py

```diff
@@ -269,6 +270,10 @@ def system(session):
 
     install_systemtest_dependencies(session, "-c", constraints_path)
 
+    # If POSTGRESQL tests and Emulator, skip the tests
+    if os.environ.get("SPANNER_EMULATOR_HOST") and database_dialect == "POSTGRESQL":
+        session.skip("Postgresql is not supported by Emulator yet.")
```

Comment:
This `if` can be moved upper to avoid checking folders in case of PostgreSQL and emulator. Not a significant thing though